lumicks/pylake/scan.py

```diff
@@ -62,6 +62,16 @@ def __getitem__(self, item):
 
         return new_scan
 
+    @property
+    def pixel_time_seconds(self):
+        """Pixel dwell time in seconds"""
+        indices = np.zeros(self.timestamps.ndim, dtype=int)
+
+        # We want the difference along the fast axis. The first (optional) index corresponds to the
+        # frame. The last two indices correspond to the imaging axes.
+        indices[-2 if self._metadata.scan_order[0] > self._metadata.scan_order[1] else -1] = 1
+        return (self.timestamps.item(tuple(indices)) - self.timestamps.item(0)) / 1e9
```

Comment:
never knew about `np.ndarray.item` ... neat

Reply:
yeah, it can be pretty useful at times :)